src/Auth/StringMomentoTokenProvider.php

```diff
@@ -14,13 +14,18 @@ class StringMomentoTokenProvider extends CredentialProvider
     protected string  $authToken;
     protected ?string $controlEndpoint = null;
     protected ?string $cacheEndpoint = null;
+    protected ?string $storageEndpoint = null;
     protected ?string $trustedControlEndpointCertificateName = null;
     protected ?string $trustedCacheEndpointCertificateName = null;
 
     public function __construct(
         string  $authToken,
         ?string $controlEndpoint = null,
         ?string $cacheEndpoint = null,
+        // TODO: adding this arg would be a breaking change for anyone currently passing in
+        //  endpointCertificateName arguments. I am pretty sure that is 0 people, but wanted
+        //  to call it out.
+        ?string $storageEndpoint = null,
```

Comment:
Raising a flag here, though as far as I know this wouldn't have any customer impact.

Reply:
I doubt anyone is building this directly as you said.